### ISSUE TITLE
apm update /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found 

### ISSUE THREAD

**Issue by mingjunyang** [atom/atom#6465]
my os is openSUSE 13.2 Gnome 3.14.0
Linux mycomputename 3.16.7-13-desktop #1 SMP PREEMPT Wed Mar 18 17:31:15 UTC 2015 (ba2afab) x86_64 x86_64 x86_64 GNU/Linux.
when i run ```apm update```
```shell
apm update

/usr/share/atom/resources/app/apm/node_modules/asar-require/node_modules/asar/node_modules/chromium-pickle/node_modules/bindings/bindings.js:83
        throw e
              ^
Error: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /usr/share/atom/resources/app/apm/node_modules/asar-require/node_modules/asar/node_modules/chromium-pickle/build/Release/pickle.node)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at bindings (/usr/share/atom/resources/app/apm/node_modules/asar-require/node_modules/asar/node_modules/chromium-pickle/node_modules/bindings/bindings.js:76:44)
    at Object.<anonymous> (/usr/share/atom/resources/app/apm/node_modules/asar-require/node_modules/asar/node_modules/chromium-pickle/lib/main.js:1:99)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

I think I update my glibc will be problem....
how to fix this?
my glibcxx
```shell
 strings /usr/lib/libstdc++.so.6 | grep GLIBCXX
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_DEBUG_MESSAGE_LENGTH
```

**Comment by mingjunyang** [comment]
非常抱歉，我又忘记告诉你们我用的哪个版本的atom了。
Soory, I forget tell you something.
```shell
atom --version
0.193.0

```

**Comment by zcbenz** [comment]
@kevinsawicki The Linux build machine uses a relatively new distribution which has a newer libc version which is not available on openSUSE 13.2.

I think I should remove the dependency of `chromium-pickle` in `asar` so we don't relay on native modules when using apm.

**Comment by kevinsawicki** [comment]
> I think I should remove the dependency of chromium-pickle in asar so we don't relay on native modules when using apm.

Sounds good :+1:

**Comment by hermanbergwerf** [comment]
+1

**Comment by zcbenz** [comment]
Fixed by https://github.com/atom/asar/pull/28.

**Comment by mingjunyang** [comment]
waiting release!坐等发布！

**Comment by guhcampos** [comment]
+1 on this issue on CentOS 7.1.1503

**Comment by Otterpocket** [comment]
@guhcampos This should be fixed in the latest release, what version of atom are you using?

**Comment by guhcampos** [comment]
@Otterpocket Version 0.194.0!

Thanks for the info. Let me know if you need help testing.

**Comment by Otterpocket** [comment]
@guhcampos This issue is fixed in 0.195.0 and 0.196.0, try installing a newer package https://github.com/atom/atom/releases

**Comment by guhcampos** [comment]
Confirmed working alright on 0.196.0 running on CentOS 7.1.1503, RPM install.

Sorry for the confusion. I keep my box updated via Yum and forgot I had installed atom with "localinstall"
